aiida_testing/mock_code/_cli.py

```diff
@@ -51,8 +64,8 @@ def run() -> None:
                 continue
             os.makedirs(os.path.join(res_dir, dirname), exist_ok=True)
             for filename in filenames:
-                if filename in ignore_files:
-                    continue
+                if check_expression(filename, ignore_files):
```

Comment:
This could be done simpler:

```
if any(fnmatch.fnmatchcase(filename, expr) for expr in ignore_files):
```

Reply:
Sure, you are right. The increased computational complexity does not matter in that case.
I would go with ```fnmatch.fnmatch``` instead of ```fnmatch.fnmatchcase``` since this works on Windows, for unix and mac it does not matter, see 'https://docs.python.org/3.5/library/os.path.html#os.path.normcase'

Reply:
You're right, `fnmatch` is probably the better choice.

Just out of curiosity, how is the computational complexity different? AFAIK, ``any`` on an iterator aborts when the first ``True`` is encountered.